tests/unit/test_handler_service.py

```diff
@@ -84,47 +85,52 @@ def test_handle(inference_handler):
 
 @require_torch
 def test_load(inference_handler):
+    context = Mock()
     with tempfile.TemporaryDirectory() as tmpdirname:
         storage_folder = _load_model_from_hub(
             model_id=MODEL,
             model_dir=tmpdirname,
         )
         # test with automatic infer
-        hf_pipeline_without_task = inference_handler.load(storage_folder)
+        hf_pipeline_without_task = inference_handler.load(storage_folder, context)
```

Comment:
Can you also add a test case to cover the use case when user doesn't provide context parameter? Just to make sure we don't break existing users.

Reply:
Updated the PR to test for both with and without context cases.